e2e/vite/src/vite-legacy.test.ts

```diff
@@ -78,40 +77,4 @@ describe('@nx/vite/plugin', () => {
       }
     });
   });
-
-  describe('react with vitest only', () => {
```

Comment:
Not needed for e2e. This can be checked within unit tests, and is indeed being checked in unit tests.